test/comparator/test_field_comparator.py

```diff
@@ -294,7 +294,7 @@ def test_resource_reference_addition_breaking(self):
             "A resource reference option is added to the field `Test`, but it is not defined anywhere",
         )
 
-    def test_resource_reference_addition_non_breaking(self):
+    def test_resource_reference_addition_non_breaking1(self):
```

Comment:
If we change the test name of the other test, this name can stay the same
```suggestion
    def test_resource_reference_addition_non_breaking(self):
```

Reply:
Done.